package.json

```diff
@@ -4,19 +4,33 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "serve": "./node_modules/.bin/http-server dist/ --cors"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "classnames": "^2.2.5",
     "react": "^15.2.1",
     "react-dom": "^15.2.1",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.6.0",
+    "react-thunk": "^1.0.0",
+    "redux-thunk": "^2.1.0",
     "typescript": "^1.8.10",
     "typings": "^1.3.1",
-    "webpack": "^1.13.1"
+    "webpack": "^1.13.1",
+    "postcss": "^5.0.21",
+    "postcss-loader": "^0.9.1",
+    "postcss-cssnext": "^2.7.0",
+    "postcss-import": "^8.1.2",
+    "style-loader": "^0.13.1",
+    "css-loader": "^0.23.1"    
```

Comment:
Probably only the following should be `dependencies`. The rest are `devDependencies`.

```
"classnames": "^2.2.5",
"react": "^15.2.1",
"react-dom": "^15.2.1",
"react-redux": "^4.4.5",
"react-router": "^2.6.0",
"react-thunk": "^1.0.0",
"redux-thunk": "^2.1.0",
```